src/main/java/tech/jhipster/lite/module/infrastructure/secondary/FileSystemSpringDockerComposeIntegrationHandler.java

```diff
@@ -0,0 +1,28 @@
+package tech.jhipster.lite.module.infrastructure.secondary;
+
+import java.nio.file.Path;
+import java.util.function.Consumer;
+import tech.jhipster.lite.module.domain.properties.JHipsterProjectFolder;
+import tech.jhipster.lite.module.domain.standalonedocker.JHipsterModuleComposeFiles;
+import tech.jhipster.lite.module.domain.startupcommand.DockerComposeFile;
+import tech.jhipster.lite.shared.error.domain.Assert;
+
+class FileSystemSpringDockerComposeIntegrationHandler {
+
+  public static final String COMPOSE_FILE_NAME = "compose.yaml";
```

Comment:
I would go for `docker-compose.yaml`, for the same reason as mentioned earlier: for explicitness.
```suggestion
  public static final String COMPOSE_FILE_NAME = "docker-compose.yml";
```